pages.pt_PT/android/wm.md

```diff
@@ -0,0 +1,13 @@
+# wm
+
+> Exibe informações da tela de um dispositivo Android
+> Esse comando só pode ser usado através da `adb shell`.
+> Mais informações: <https://adbinstaller.com/commands/adb-shell-wm-5b672b17e7958178a2955538>.
+
+- Mostrar o tamanho da tela de um dispositivo Android:
+
+`wm {{size}}`
+
+- Mostrar a densidade de pixeis da tela de um dispositivo Android:
```

Comment:
Em português, "pixeis" (com i) tem acento (ver [aqui](https://translatewiki.net/wiki/Portal:Pt/Cheatsheet/pt#pixels) e [aqui](https://pt.wiktionary.org/wiki/p%C3%ADxel)):

```suggestion
- Mostrar a densidade de píxeis da tela de um dispositivo Android:
```

Reply:
My bad!

Reply:
Not at all, that's a common confusion since it's a loanword :)